include/script_pokemon_util.h

```diff
@@ -6,14 +6,14 @@
 
 BOOL GiveMon(HeapID heapId, SaveData *saveData, int species, int level, int form, u8 ability, u16 heldItem, int ball, int encounterType);
 BOOL GiveEgg(HeapID heapId, SaveData *saveData, int species, u8 metLocation, MapsecType sp20, int sp24);
-void PartyMonSetMoveInSlot(PARTY *party, int mon_slot, int move_slot, u16 moveId);
-int GetIdxOfFirstPartyMonWithMove(PARTY *party, u16 move);
-int CountAlivePokemon(PARTY *party);
-Pokemon *GetFirstAliveMonInParty_CrashIfNone(PARTY *party);
-u8 GetIdxOfFirstAliveMonInParty_CrashIfNone(PARTY *party);
-Pokemon *GetFirstNonEggInParty(PARTY *party);
-BOOL HasEnoughAlivePokemonForDoubleBattle(PARTY *party);
-BOOL ApplyPoisonStep(PARTY *party, u16 location);
+void PartyMonSetMoveInSlot(Party *party, int mon_slot, int move_slot, u16 moveId);
+int GetIdxOfFirstPartyMonWithMove(Party *party, u16 move);
+int CountAlivePokemon(Party *party);
```

Comment:
Party_ prefix in the functions in this file

Reply:
out of scope